src/coreclr/jit/hwintrinsiclistarm64sve.h

```diff
@@ -16,6 +16,41 @@
+
+
+
+// ***************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************
+//                 ISA            Function name                                              SIMD size  NumArg  EncodesExtraTypeArg                                                                                            Instructions                                                                                        Category                           Flags
+//                                                                                                                          {TYP_BYTE,           TYP_UBYTE,          TYP_SHORT,          TYP_USHORT,         TYP_INT,            TYP_UINT,           TYP_LONG,           TYP_ULONG,          TYP_FLOAT,          TYP_DOUBLE}
+// ***************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************
+// Special intrinsics that are generated during importing or lowering
+
+HARDWARE_INTRINSIC(Sve,           ConvertMaskToVector,                                              -1,      1,      true,  {INS_sve_mov,        INS_sve_mov,        INS_sve_mov,        INS_sve_mov,        INS_sve_mov,        INS_sve_mov,        INS_sve_mov,        INS_sve_mov,        INS_sve_mov,        INS_sve_mov},     HW_Category_Helper,                HW_Flag_Scalable|HW_Flag_MaskedOperation)
+HARDWARE_INTRINSIC(Sve,           ConvertVectorToMask,                                              -1,      2,      true,  {INS_sve_cmpne,      INS_sve_cmpne,      INS_sve_cmpne,      INS_sve_cmpne,      INS_sve_cmpne,      INS_sve_cmpne,      INS_sve_cmpne,      INS_sve_cmpne,      INS_sve_cmpne,      INS_sve_cmpne},   HW_Category_Helper,                HW_Flag_Scalable|HW_Flag_ReturnsPerElementMask|HW_Flag_LowMaskedOperation)
+
+HARDWARE_INTRINSIC(Sve,           CreateTrueMaskAllByte,                                            -1,      0,      false, {INS_invalid,        INS_sve_ptrue,      INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid},     HW_Category_Helper,           	  HW_Flag_Scalable|HW_Flag_ReturnsPerElementMask)
```

Comment:
Why can't we have just 1 entry of `CreateTrueMaskAll`?

Reply:
Without changing any other code, if I add the following:

```
// HARDWARE_INTRINSIC(Sve,           CreateTrueMaskAll,                                                -1,      0,      false, {INS_sve_ptrue,      INS_sve_ptrue,      INS_sve_ptrue,      INS_sve_ptrue,      INS_sve_ptrue,      INS_sve_ptrue,      INS_sve_ptrue,      INS_sve_ptrue,      INS_sve_ptrue,      INS_sve_ptrue},   HW_Category_Helper,           	  HW_Flag_Scalable|HW_Flag_ReturnsPerElementMask)
```

Then I get a segfault during build. This is because a hwintrinsic with 0 args cannot have multiple types as the return argument is the only way to determine it's signature.

But, this intrinsic is internal and is not exposed to C#. So, I can fudge this by setting `EncodesExtraTypeArg` to `true` in the table and that gets rid of the segfault. That won't break anything, but it feels wrong.

Thoughts?

Reply:
>  EncodesExtraTypeArg to true in the table and that gets rid of the segfault

We do that for other internal intrinsics, so should be fine, unless Tanner has an opinion.

Reply:
Done. I had to set number of args to -1, otherwise asserts fire.